src/login.js

```diff
@@ -1240,7 +1235,7 @@ async function F_LOGIN() {
 	});
 	WM.events.on({
 		"moveend": onMoveEnd,
-		"zoomend": HLAllObjects,
+		"zoomend": delayForceHLAllObjects,
```

Comment:
Why `HLAllObjects` is not enough?

Reply:
Features may still be loading after the zoom end event, so we want to use the same check for loadingFeatures being false or delay if not.